test/unit/Signer.test.js

```diff
@@ -6,9 +6,9 @@ const { StreamMessage, StreamMessageV30, StreamMessageV29 } = MessageLayer
 
 describe('Signer', () => {
     describe('construction', () => {
-        it('should sign when constructed with private key', () => {
+        it('should sign when constructed with private key', async () => {
             const signer = new Signer({
-                privateKey: '348ce564d427a3311b6536bbcff9390d69395b06ed6c486954e971d960fe8709',
+                privateKey: '0x348ce564d427a3311b6536bbcff9390d69395b06ed6c486954e971d960fe8709',
```

Comment:
Private keys passed to the client should work both with `0x` prefix and without. There hardly is a standard representation for private keys - in my experience without the prefix is more common. Please include test cases both with and without the prefix.